src/main/java/org/mule/extension/http/api/HttpRequestAttributes.java

```diff
@@ -130,4 +130,17 @@ public Certificate getClientCertificate() {
     return clientCertificate;
   }
 
+  public String toString() {
+    StringBuilder builder = new StringBuilder();
+    builder.append("The requester has these values:\n");
+    builder.append("Request Headers:\n");
+    headers.entrySet().stream().forEach(header -> builder.append("   " + header.getKey() + ": " + header.getValue() + "\n"));
```

Comment:
Why are you not using the builder here as well and concatenating as usual?

Reply:
Make the separator a constant to reuse it (the `"    "`). You can define it higher up in the attributes hierarchy.